renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "schedule": [
+    "before 5am on Monday"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["patch"],
+      "groupName": "all non-major dependency bump",
+      "groupSlug": "all-patch",
+      "automerge": true
+    },
+    {
+      "extends": "packages:linters",
+      "groupName": "linters",
+      "automerge": true
+    },
+    {
+      "extends": "packages:postcss",
+      "groupName": "postcss packages"
+    },
+    {
+      "extends": "packages:test",
+      "groupName": "test packages",
+      "automerge": true
+    },
+    {
+      "groupName": "definitelyTyped",
+      "matchPackagePrefixes": ["@types/"],
+      "automerge": true
+    },
+    {
+      "matchPackageNames": ["swc-plugin-coverage-instrument"],
+      "enabled": false
+    }
```

```suggestion
    {
      "matchPackageNames": ["swc-plugin-coverage-instrument"],
      "enabled": false
    },
    {
      "enabled": false,
      "matchDepTypes": [
        "peerDependencies",
        "engines"
      ]
    }
```

As mentioned earlier - disable Node.js & `peerDependencies` upgrades.

---

Okay, here - I would not prevent renovate from rising PRs with peerDeps upgrades. IMO it might be useful to get them just to be reminded that they probably should be updated soon 😄 
I agree on disabling `engines` upgrades, though. Do we have an agreement? 😄

---

Yep, it seems good to me ;)

---

28441c372fcba11fb8d7589a037d640431ebf18d